listenbrainz/webserver/views/api.py

```diff
@@ -76,6 +76,9 @@ def submit_listen():
     if mb_engine and current_app.config["REJECT_LISTENS_WITHOUT_USER_EMAIL"] and not user["email"]:
         raise APIUnauthorized(REJECT_LISTENS_WITHOUT_EMAIL_ERROR)
 
+    if user['is_paused']:
+        raise APIUnauthorized("user_id is paused.")
```

Comment:
" ... is paused and is currently not accepting listens. Feel free to [Contact Us] if you have any questions about this."